custom_components/media_player_template/media_player.py

```diff
@@ -1,4 +1,7 @@
-"""Support for switches which integrates with other components."""
+"""
+Template for media-player
+https://github.com/Sennevds/media_player.template
+"""
```

Comment:
The old description seemed to be copied from somewhere else, I assumed you would like the same description as in `__init__.py